tests/test_sampling.py

```diff
@@ -212,7 +250,7 @@ def logprob_fn(x):
 
     def mcse_test(self, samples, true_param, p_val=0.01):
         posterior_mean = jnp.mean(samples, axis=[0, 1])
-        ess = diagnostics.effective_sample_size(samples, chain_axis=1, sample_axis=0)
+        ess = diagnostics.effective_sample_size(samples, chain_axis=0, sample_axis=1)
```

Comment:
There was an error on the original test_sampling.py on which was the axis of chain and which of samples. Also change in line 289.